### PR TITLE
feat: update 'ykchorus' to version 0.2.3'

### PR DESCRIPTION
**Pull request by SpotlightKid**

### CODE REVIEW

File: packages/ykchorus/PKGBUILD [hunk 1]
```diff
@@ -18,29 +18,27 @@ optdepends=(
   'vst-host: for VST plugin'
 )
 groups=(ladspa-plugins lv2-plugins vst-plugins)
```
Comment (cbix) [inline packages/ykchorus/PKGBUILD:20]:
```suggestion
groups=(pro-audio ladspa-plugins lv2-plugins vst-plugins)
```

File: packages/ykchorus/PKGBUILD [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 pkgname=ykchorus
 _plugin_uri="https://chrisarndt.de/plugins/$pkgname"
-pkgver=0.2.2
+pkgver=0.2.3
 pkgrel=1
 pkgdesc='A stereo dual-chorus audio effect LV2, LADSPA and VST2 plugin and JACK application'
 arch=(x86_64)
```
Comment (cbix) [inline packages/ykchorus/PKGBUILD:9]:
```suggestion
arch=(x86_64 aarch64)
```

File: packages/ykchorus/PKGBUILD [hunk 3]
```diff
@@ -18,29 +18,27 @@ optdepends=(
   'vst-host: for VST plugin'
```
Comment (cbix) [inline packages/ykchorus/PKGBUILD:18]:
```diff
- 'ladspa-host: for LV2 plugin'
+ 'ladspa-host: for LADSPA plugin'
```

File: packages/ykchorus/PKGBUILD (hunk 2 above)
Comment (SpotlightKid) [reply packages/ykchorus/PKGBUILD]: Done in ee6538c.

File: packages/ykchorus/PKGBUILD (hunk 1 above)
Comment (SpotlightKid) [reply packages/ykchorus/PKGBUILD]: Done in ffc2840.

File: packages/ykchorus/PKGBUILD (hunk 3 above)
Comment (SpotlightKid) [reply packages/ykchorus/PKGBUILD:18]: Done in 7df9a45.

**Review by cbix — APPROVED** [review]
Nice to see your nvchecker cronjob working ;)

**Pull request merged**